packages/frontend/src/retro/context/RetroContext.tsx

```diff
@@ -255,6 +278,10 @@ export function RetroContextProvider(props: RetroContextProviderProps) {
     handleAcceptJoinUser: acceptJoinUser,
     handleAddToWaitingList,
     handleIsVotingEnabledChanged,
+    handleStartTimer,
+    handlePauseTimer,
+    handleStopTimer,
```

Comment:
Is there a whitespace too much or is my phones formatting wrong?
Will recheck on pc